pkg/database/migrations/postgres/5_edits.up.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE "edits" (
+  "id" uuid not null primary key,
+  "user_id" uuid not null,
+  "operation" varchar(10) not null,
+  "target_type" varchar(10) not null,
+  "data" jsonb,
+  "votes" integer not null default 0,
+  "status" varchar(20) not null,
+  "applied" boolean default FALSE not null,
+  "created_at" timestamp not null,
+  "updated_at" timestamp not null,
+  foreign key("user_id") references "users"("id")
```

Comment:
I think this might need some further thought. Need to consider that users may be removed.

Reply:
Good point. I think it should suffice to make the column nullable, and set `ON DELETE SET NULL`, so users can be freely deleted without taking edit history down with them.